README.md

```diff
@@ -1,132 +1,37 @@
 Chitter Challenge
 =================
 
-* Feel free to use Google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday morning
+In this project, I have created a mock twitter called chitter where users can post messages and see them displayed. They are also able to see the messages posted by other users.
 
-Challenge:
--------
+## Setting up your database
 
-As usual please start by forking this repo.
+**Create Chitter DB**
 
-We are going to write a small Twitter clone that will allow the users to post messages to a public stream.
+1. Connect to ```psql``` in your terminal.
+2. Create the database using the psql command ```CREATE DATABASE chitter;```
+3. Connect to the database using the pqsl command ```\c chitter;```
+4. Run the query we have saved in the file ```01_create_peeps_table.sql```
 
-Features:
--------
+**Create Chitter TEST DB**
 
-```
-STRAIGHT UP
+Do the above again, with the following changes:
 
-As a Maker
-So that I can let people know what I am doing  
-I want to post a message (peep) to chitter
+- Step 2: ```CREATE DATABASE chitter_test;```
+- Step 3: ``` \c chitter_test;```
 
-As a maker
-So that I can see what others are saying  
-I want to see all peeps in reverse chronological order
+**Adding time to the databases**
```

Comment:
Can you refactor the above to include time in one step, rather than running two queries? I haven't looked further along in your code yet, but could you implement a 'timestamp' in SQL, so that the table will automatically give it a time?